### PR TITLE
Card Atom

### PR DESCRIPTION
**Pull request by gankoji**

### CODE REVIEW

File: clients/web/src/lib/components/atoms/Card/Card.story.svelte [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+    import type {Hst as _Hst} from "@histoire/plugin-svelte";
+    import Card from "./Card.svelte";
+
+    type sizes = "small" | "default" | "large" | "extraLarge";
+
+    const sizeOptions = ["small", "default", "large", "extraLarge"];
+
```
Comment (0c370t) [inline clients/web/src/lib/components/atoms/Card/Card.story.svelte:8]: Is this dead code?

File: clients/web/src/lib/components/atoms/Card/Card.svelte [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                viewBox="0 0 20 20"
+                xmlns="http://www.w3.org/2000/svg"
+                ><path
+                    fill-rule="evenodd"
+                    d="M10.293 3.293a1 1 0 011.414 0l6 6a1 1 0 010 1.414l-6 6a1 1 0 01-1.414-1.414L14.586 11H3a1 1 0 110-2h11.586l-4.293-4.293a1 1 0 010-1.414z"
+                    clip-rule="evenodd"
+                /></svg
+            >
+        </a>
+    {/if}
```
Comment (0c370t) [inline clients/web/src/lib/components/atoms/Card/Card.svelte:44]: We can likely rip some (or all) of this out and just replace it with a `<slot/>` (or 2, one for the header if we feel it's needed); to keep this a little more agnostic

**Review by 0c370t — CHANGES_REQUESTED** [review]

File: clients/web/src/lib/components/atoms/Card/Card.story.svelte (hunk 1 above)
Comment (gankoji) [reply clients/web/src/lib/components/atoms/Card/Card.story.svelte:8]: Yes.

File: clients/web/src/lib/components/atoms/Card/Card.svelte (hunk 2 above)
Comment (gankoji) [reply clients/web/src/lib/components/atoms/Card/Card.svelte]: Done.

**Review by 0c370t — APPROVED** [review]

**Pull request merged**